### PR TITLE
Responsive charts with width/height parameter

### PR DESCRIPTION
**Pull request by Langleu**
Added responsivness for pie-chart and bar-chart.
Width and height attribute can be given otherwise it will just adjust to the parent element

### CODE REVIEW

File: barchart-basic.html [hunk 1]
```diff
@@ -74,15 +131,12 @@
...
       },
-      /**
-      * Normalizes the data supplied by the API to work with Plotly
-      *
-      * @param {Object} data The data (as JSON) to be normalized
-      * @return {Array} The normalized data
-      */
-      _normalizeJSON: function (data) {
+
+      _readDataFromResponse: function (data) {
```
Comment (FWirtz) [inline barchart-basic.html]: please document

File: barchart-basic.html [hunk 2]
```diff
@@ -74,15 +131,12 @@
...
+        console.log("y");
```
Comment (FWirtz) [inline barchart-basic.html]: omit `console.log` for merge

**Comment by Langleu** [comment]
👍

**Comment by FWirtz** [comment]
:+1:

**Pull request merged**